go.mod

```diff
@@ -0,0 +1,21 @@
+module github.com/giantswarm/k8scloudconfig
+
+go 1.13
+
+require (
+	github.com/Masterminds/semver v1.5.0
+	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32 // indirect
+	github.com/giantswarm/apiextensions v0.1.0
+	github.com/giantswarm/microerror v0.1.0
+	github.com/gogo/protobuf v1.3.1 // indirect
+	github.com/juju/errgo v0.0.0-20140925100237-08cceb5d0b53 // indirect
+	github.com/stretchr/testify v1.4.0
+	golang.org/x/net v0.0.0-20191207000613-e7e4b65ae663 // indirect
+	gopkg.in/inf.v0 v0.9.1 // indirect
+	gopkg.in/yaml.v2 v2.2.8
+	k8s.io/apiextensions-apiserver v0.0.0-20191114105449-027877536833 // indirect
```

Comment:
The k8s deps need to be at least on `v0.16.6`.

Reply:
After updating all to 0.2.0 for gs libs, it's gone.